.travis.yml

```diff
@@ -19,8 +19,11 @@ env:
   # Ensure that we build without warnings.
   - CARGO_FLAGS="--features 'strict'"
 
+before_script: (cargo install rustfmt || true)
```

Comment:
From http://johannh.me/blog/rustfmt-ci.html:
> Since cargo returns an error if a package is installed already, I have to use (cargo install rustfmt || true).

However if `cargo install rustfmt` fails without installing it et all, the build will fail.